server.go

```diff
@@ -59,6 +73,59 @@ func hacker_handler(w http.ResponseWriter, r *http.Request) {
   w.Write(js)
 }
 
-func today(h string) bool{
+func (api *Api) CreateHackerHandler(w http.ResponseWriter, r *http.Request) {
+  err := r.ParseForm()
+  if err != nil {
+    http.Error(w, err.Error(), http.StatusInternalServerError)
+    return
+  }
+
+  // save data
+  hacker := Hacker{Name: r.Form.Get("name")}
+  if err := api.DB.Where(Hacker{Name: r.Form.Get("name")}).FirstOrCreate(&hacker).Error; err != nil {
+    http.Error(w, err.Error(), http.StatusInternalServerError)
+    return
+  }
+
+  // make JSON
+  js, err := json.Marshal(&hacker)
+  if err != nil {
+    http.Error(w, err.Error(), http.StatusInternalServerError)
+    return
+  }
+
+  // return JSON
+  w.Header().Set("Content-Type", "application/json")
+  w.Write(js)
+}
+
+func today(h string) bool {
   return false
 }
+
+func InitEnv() {
```

Comment:
I don't understand what this does. Is it just checking to see if it can access the environment?